src/docs/pages/getting-started.md

```diff
@@ -36,6 +36,8 @@ import { LdButton } from '@emdgroup-liquid/liquid/dist/components/ld-button'
 customElements.define('ld-button', LdButton)
 ```
 
+> **Note**: Child components are auto-imported and defined recursively. However, components used in slots still need to be imported and defined.
+
```

Comment:
```suggestion
```
I would remove that as this does not provide any necessary information to the user (if he just did not think about this, the components would just work as he'd expect and he doesn't know in which components we use `ld-*` components in the shadow DOM) and rather leads to confusion than it helps.